upup/pkg/fi/cloudup/awstasks/launchtemplate_target_api.go

```diff
@@ -164,8 +166,19 @@ func (t *LaunchTemplate) RenderAWS(c *awsup.AWSAPITarget, a, e, changes *LaunchT
 			LaunchTemplateName: t.Name,
 			LaunchTemplateData: data,
 		}
-		if _, err = c.Cloud.EC2().CreateLaunchTemplateVersion(input); err != nil {
+		if version, err := c.Cloud.EC2().CreateLaunchTemplateVersion(input); err != nil {
 			return fmt.Errorf("error creating LaunchTemplateVersion: %v", err)
+		} else {
```

Comment:
Please keep the same style.
```suggestion
		version, err := c.Cloud.EC2().CreateLaunchTemplateVersion(input)
		if err != nil {
			return fmt.Errorf("error creating LaunchTemplateVersion: %v", err)
		}
		...
```

Reply:
I really dislike the `if err {} else {}` pattern, it's hard to follow, but can be changed later.

Reply:
Concur. There should not be an `else` after an if block that always returns.